### PR TITLE
add prod yaml

### PR DESCRIPTION
**Pull request by bellatoris**

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #109** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/109" target="_blank">Graphite</a> 
    * **PR #110** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/110" target="_blank">Graphite</a>  👈
      * **PR #111** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/111" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/110?utm_source=stack-comment).

File: infra/helm/scc-admin-frontend/values-prod.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: public.ecr.aws/q0g6g7m8/scc-admin-frontend
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest-rc"
```
Comment (Zeniuus) [inline infra/helm/scc-admin-frontend/values-prod.yaml:11]:
```suggestion
  tag: "latest"
```
??

Comment (Zeniuus) [reply infra/helm/scc-admin-frontend/values-prod.yaml:11]: 이미지는 푸시해뒀습니당

**Review by Zeniuus — APPROVED** [review]

File: infra/helm/scc-admin-frontend/values-prod.yaml (hunk 1 above)
Comment (bellatoris) [reply infra/helm/scc-admin-frontend/values-prod.yaml:11]: 기존 이미지가 rc더라구요

Comment (Zeniuus) [reply infra/helm/scc-admin-frontend/values-prod.yaml:11]: latest도 푸시해두었습니당

**Comment by bellatoris** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/110?utm_source=stack-comment).


Job Reference: XhQAUDpWA2PpvPF9lhCY

**Comment by bellatoris** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/110?utm_source=stack-comment).


Job Reference: PGoqsX4Hok2WdG0O9tTv

**Pull request merged**